other/materials_designer/specific_examples/defect_creation_point_substitution_graphene.ipynb

```diff
@@ -0,0 +1,292 @@
+{
```

Comment:
<p>1.3. Load input material</p><p> </p><p>Explain that the material comes from a set of test fixtures</p><p> </p><p>Create a task to update Standata and use it here instead</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/147/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>